examples/tutorials/03_vision/04_exposure.py

```diff
@@ -46,24 +46,26 @@ def camera_info(image, scale, annotator=None, world=None, **kw):
     d = ImageDraw.Draw(image)
     bounds = [3, 0, image.width, image.height]
 
-    camera_config = world.robot.camera_config
+    cam = world.robot.camera
```

Comment:
Maybe rename `cam` to `camera`: it is clearer and translatable for non-English speakers.